templates/assignee_no_login.html

```diff
@@ -1,4 +1,6 @@
-<p>The following {{ plural('bug has', data, pword='bugs have') }} an assignee who is inactive on Bugzilla:</p>
+<p>
+    The following {{ plural('bug has', data, pword='bugs have') }} been unassigned due to assignee inactivity on Bugzilla:
+</p>
```

Comment:
Could you please minimize the diff here?

Reply:
Done in 0a65348350001c3222ef33d3ca60d799af0ad400. The auto-linting reformatted the tags to be on different lines than the actual text, so I made the message a little shorter.